cli/scripts/export_test_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+TEST_ENV_FILE=".test.env"
+
+# Check if the .env file exists
+if [ ! -f "$TEST_ENV_FILE" ]; then
+    echo "$TEST_ENV_FILE does not exist."
+    exit 1
+fi
+
```

Comment:
this is just for local development. In the CI/CD we're fetching envs from GH secrets

Reply:
the main workflow i'm thinking of here is that we can just pass around a `.test.env` file and then paste it in the cli root directory instead of manually exporting the secrets in the local env